README.md

```diff
@@ -1 +1,3 @@
-# hello-microverse
+# Hello Microverse
+
+My first project at Microverse.
```

Comment:
- [ ] Kindly Make the readme file more descriptive and professional. You can see example [here](https://github.com/microverseinc/readme-template)

Reply:
Thank you @rotimiazeez .

Reply:
Hi @rotimiazeez, with the changes I made on the Readme.MD file did I pass the Code review process? If not then please give me instructions on what to do?